pages/login.js

```diff
@@ -145,14 +145,9 @@ const Login = () => {
         }
     };
 
-    const [
-        login,
-        {
-            data: data,
-            loading: loading,
-            error: error,
-        },
-    ] = useMutation(LOGIN_USER, { client })
+    const [login, { data: data, loading: loading, error: error }] = useMutation(LOGIN_USER, {
+        client
+    });
```

Comment:
@Souptik2001 Let's break it into multiple lines to make it more readable

Reply:
@imranhsayed As per the current prettier configuration, as this line doesn't exceed a threshold line length limit, that's why it is showing error if we are breaking this line.

So, should we change that configuration, because otherwise there will be linting error.

Reply:
Yes please. let's update the configuration to make it look more readable @Souptik2001